src/index.js

```diff
@@ -1,51 +1,60 @@
 const DEFAULT_ENVIRONMENTS = [
   {
-    displayName: 'local',
-    host: /(^localhost(:[0-9]+)?$|\.test$)/,
+    displayName: "local",
+    host: /(^localhost(:[0-9]+)?$|\.test$)/
   },
   {
-    displayName: 'dev',
+    displayName: "dev",
```

Comment:
If we're going to start formatting this repository with Prettier (which I'm totally on board with!), let's set our code style settings in `package.json` (like [here](https://github.com/DoSomething/phoenix-next/blob/95b64698172c1e4a7f14c0945ceedf17d528f1bb/package.json#L29-L32)).

Reply:
Let's also install Prettier in this project and configure the [pre-commit hook](https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick) so that we maintain these formatting rules whether or not contributors have the editor extension.

Reply:
All set! This has now happened in #4 and will be rebased from master into this PR :)